docs/Using-Fleet/Permissions.md

```diff
@@ -43,10 +43,10 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 | Filter hosts by software                                                                                                                   | ✅        | ✅         | ✅          | ✅     |        |
 | Filter software by team\*                                                                                                                  | ✅        | ✅         | ✅          | ✅     |        |
 | Manage [vulnerability automations](https://fleetdm.com/docs/using-fleet/automations#vulnerability-automations)                             |          |           |            | ✅     | ✅      |
-| Run only designated, **observer can run** ,queries as live queries against all hosts                                                       | ✅        | ✅         | ✅          | ✅     |        |
+| View and run queries marked as **observer can run** against all hosts                                                                      | ✅        | ✅         | ✅          | ✅     |        |
 | Run any query as [live query](https://fleetdm.com/docs/using-fleet/fleet-ui#run-a-query) against all hosts                                 |          | ✅         | ✅          | ✅     |        |
 | Create, edit, and delete queries                                                                                                           |          |           | ✅          | ✅     | ✅      |
-| View all queries                                                                                                                           | ✅        | ✅         | ✅          | ✅     |        |
+| View all queries                                                                                                                           |         |          | ✅          | ✅     |        |
```

Comment:
This would contradict the API, what about a clarification somewhere that observers can view all queries but the UI/fleetctl filter out queries they cannot run.

Reply:
@zhumo Here's my attempt: https://github.com/fleetdm/fleet/pull/11251

Let me know if it makes sense.

Reply:
@zhumo https://github.com/fleetdm/fleet/pull/11251 has been merged to `main` (see current docs: https://fleetdm.com/docs/using-fleet/permissions)
![Screenshot 2023-04-27 at 09 05 18](https://user-images.githubusercontent.com/2073526/234856919-aa78fc68-c107-4e94-b82e-01635f3fa3c2.png)


Should we close this PR or maybe propose any amends with respect to the current `main` version.

Reply:
Thanks